packages/tenant-process/src/routers/TenantRouter.ts

```diff
@@ -235,10 +236,13 @@ const tenantsRouter = (
           INTERNAL_ROLE,
         ]);
 
-        const tenant = await tenantService.getTenantById(
+        const { data: tenant, metadata } = await tenantService.getTenantById(
           unsafeBrandId(req.params.id),
           ctx
         );
+
+        setMetadataVersionHeader(res, metadata);
```

Comment:
Should be added to tenant API spec (see https://github.com/pagopa/interop-be-monorepo/pull/1849/files#diff-a50f62f3a4939e0ad8d4f443bf76522090263ad0bfabf14e174d8d83475fc372R39)